practical_09/silver_service_taxi_test.py

```diff
@@ -0,0 +1,28 @@
+"""
+Code to test the functionality of class Car() and subclass Taxi()
+"""
+
+from silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    test_taxi_class()
+
+
+def test_taxi_class():
+    """Demo test code to show how to use SilverServiceTaxi class."""
+
+    "Test __str__ method"
```

Comment:
I haven't seen this before. You're using ("") string literals as a comment. 😮
Is this a better choice than what we've been shown in class (#)?
Is this a common convention in the Python community?